service/test/integration1_test.go

```diff
@@ -371,6 +371,121 @@ func Test1ClientServer_clearBreakpoint(t *testing.T) {
 	})
 }
 
+func Test1ClientServer_disableHitCondLSSBreakpoint(t *testing.T) {
```

Comment:
This file (and the withTestClient1 helper) tests the old API. We don't add new tests for the old API anymore, the tests are only here to ensure backwards compatibility with old clients. This test (and the other one should be in integration2_test and should use withTestClient2.
It should be easy to change.